### PR TITLE
Add dry run functionality

### PR DESCRIPTION
**Pull request by henrymcconville**
Added optional dry run flag to allow proposed scripts to be run without applying

### CODE REVIEW

File: snowchange/cli.py [hunk 1]
```diff
@@ -287,7 +297,7 @@ def create_change_history_table_if_missing(change_history_table, snowflake_sessi
   query = "CREATE TABLE IF NOT EXISTS {0}.{1} (VERSION VARCHAR, DESCRIPTION VARCHAR, SCRIPT VARCHAR, SCRIPT_TYPE VARCHAR, CHECKSUM VARCHAR, EXECUTION_TIME NUMBER, STATUS VARCHAR, INSTALLED_BY VARCHAR, INSTALLED_ON TIMESTAMP_LTZ)".format(change_history_table['schema_name'], change_history_table['table_name'])
   execute_snowflake_query(change_history_table['database_name'], query, snowflake_session_parameters, autocommit, verbose)
 
-def fetch_change_history(change_history_table, snowflake_session_parameters, autocommit, verbose):
+def fetch_change_history(change_history_table, snowflake_session_parameters, autocommit, verbose, dry_run):
```
Comment (sfc-gh-jhansen) [inline snowchange/cli.py:300]: I don't see where dry_run is used in this function?

**Review by sfc-gh-jhansen — CHANGES_REQUESTED** [review]
Looks pretty good, just one clean-up item it seems. Thanks!

File: snowchange/cli.py (hunk 1 above)
Comment (henrymcconville) [reply snowchange/cli.py]: good spot, fixed 👍

**Comment by sfc-gh-jhansen** [comment]
Looks good, thanks @henrymcconville! I'm so sorry, but I forgot one thing during my last review. Can you also please update the snowchange version number (in both setup.py and cli.py) and the change log (in CHANGELOG.md). Thanks!

**Comment by henrymcconville** [comment]
@sfc-gh-jhansen done 👍

**Review by sfc-gh-jhansen — APPROVED** [review]
Looks good, thanks!

**Pull request merged**